ouroboros-consensus/changelog.d/20250130_093251_tdammers_mlocked_kes_rebase.md

```diff
@@ -0,0 +1,9 @@
+### Breaking
+
+- Use new mlocked KES API to represent KES sign keys internally. This ensures
```

Comment:
It would be good to link to the new API pr wherever it is (cardano-base?) or to a document that explains the design.